autoendpoint/src/tags.rs

```diff
@@ -92,6 +92,7 @@ impl Tags {
     }
 }
 
+/*
```

Comment:
Why commented out, is this just not used in autoendpoint?

Reply:
It's commented out because the `slog::Key::from(key.clone())` no longer produces a string that is equivalent to `static. I'd like to eventually return the function, but for now, I've gone and converted the Keys to a formatted value.

Reply:
This really confused me after seeing the `slog::Key` docs state it implements `impl From<String> for Key` -- turns out it only really [uses the "dynamic" version of `Key`](https://github.com/slog-rs/slog/blob/master/src/key/mod.rs) when the `dynamic-keys` feature flag is enabled, which we're missing. So switching it back on should let us uncomment this again.